web/styles/popovers.css

```diff
@@ -598,7 +598,7 @@ ul.popover-group-menu-member-list {
         }
 
         .stream-privacy {
-            display: inline-block;
+            display: inline-flex;
```

Comment:
We can get out of the business of `inline-flex` and friends here just by making the parent `<td>` a flexbox itself. My solution looks like this (with `gap: 3px` to make up for the space that will be lost with the removal of `inline-` display:

![Screenshot 2024-09-05 at 4 01 36 PM](https://github.com/user-attachments/assets/b6dd93c4-741f-4428-ad5c-dba968217281)

See my note below about how to better reference these elements without the nth-of-type pseudo class selectors.